### PR TITLE
compute: separate groups' name and id

### PR DESCRIPTION
**Pull request by greut**
Ref. #70, Requires https://github.com/exoscale/egoscale/pull/53

Better errors, when you try to mix id and name for both affinity or security groups.

```hcl
security_group_ids = ["${exoscale_security_group.default.name}"]
```

> * exoscale_compute.ada: API error 431 (internal code: 9999): Unable to execute API command deployvirtualmachine due to invalid value. Invalid parameter securitygroupids value=default due to incorrect long value format, or entity does not exist or due to incorrect parameter annotation for the field in api cmd class.

```hcl
security_groups = ["${exoscale_security_group.default.id}"]
```

> * exoscale_compute.ada: API error 431 (internal code: 4350): Unable to find group by name 21a14a60-be48-4bde-8509-ae6ed0231ed0

And a better output.

```
$ terraform show
exoscale_compute.ada:
  ...
  security_group_ids.# = 1
  security_group_ids.552509924 = 21a14a60-be48-4bde-8509-ae6ed0231ed0
  security_groups.# = 1
  security_groups.3814588639 = default
```

### CODE REVIEW

File: exoscale/security_group_resource.go [hunk 1]
```diff
@@ -152,8 +153,8 @@ func importSecurityGroup(d *schema.ResourceData, meta interface{}) ([]*schema.Re
 	id := d.Id()
 	name := ""
 	if !isUUID(id) {
-		id = ""
 		name = id
+		id = ""
```
Comment (greut) [inline exoscale/security_group_resource.go]: :hankey: !

**Comment by greut** [comment]
ping @nuschk, what do you think of this change?

**Comment by nuschk** [comment]
Looks good! 👍

**Comment by greut** [comment]
the CloudStack provider does the same https://www.terraform.io/docs/providers/cloudstack/r/instance.html#security_group_ids

**Pull request merged**